amr-wind/equation_systems/icns/source_terms/CoriolisForcing.cpp

```diff
@@ -36,16 +36,21 @@ CoriolisForcing::CoriolisForcing(const CFDSim& sim)
     m_sinphi = std::sin(m_latitude);
     m_cosphi = std::cos(m_latitude);
 
-    // Read the rotational time period (in seconds)
-    amrex::Real rot_time_period = 86400.0;
+    // Read the rotational time period (in seconds) -- This is 23hrs and 56
+    // minutes and 4.091 seconds
```

Comment:
I've been thinking about this a lot the past couple months.  24 hours is a solar day, and 23 hrs, 56 min, 4 s is a sidereal day, so with reference to fixed stars.  I think you are right in using it instead of the solar day.  What I am not completely sure about is that the Earth is both rotating and traveling about a curved orbit about the Sun.  Is the sidereal day the thing to use?  I think so...  Where can we find the definite answer.